### ISSUE TITLE
Flat/planar mesh

### ISSUE THREAD

**Issue by aegis1980** [JWock82/PyNite#92]
I noticed various mesh forms are subclassed in `mesh.py`, but not a simple planar mesh. 
I wondered whether there ought to be one?
Wall examples create planar mesh but planar meshes (as a class) also more broadly applicable (floors, glass/panel* analysis).
Happy to implement & make a pull request, just wanted to make sure I'd not missed something more fundamental.
Jon 

*I have an ulterior motive of trying to implement a pseudo-non-linear geometric analysis for glass. Separate feature/sub project.

**Comment by JWock82** [comment]
Meshing is a new feature I just implemented in the last couple of weeks. I started with circular meshes since I design a lot of tanks as an engineer. I do a lot of rectangular tanks as well, so a rectangular mesh is on my radar. I plan to eventually convert the wall example into a mesh. Probably one of the next features I work on. I may work on contour smoothing first. These MITC4 quadrilaterals only seem to produce center stresses without smoothing.
